documentation/FlowManager.md

```diff
@@ -90,3 +90,98 @@ OR if the last login/logout process for <mytoken> finished less than 5 seconds a
 }
 ```
 * **Notes:** Currently the payloads of the online flowmanager does **not** take contexts into account. The current payloads are simplified (and there for legacy purposes). In the future we could easily imagine that users would want the context information.
+
+#### Get lifecycle instructions from Online Flowmanager (GET /:token/untrusted-settings/:device)
+* **description**: Get lifecycle instructions, instead of actual settings, from the online flowmanager
```

Comment:
This description is unclear to me. What do we mean by "actual settings"?

Reply:
Neither of these terms is (about to be) correct. Rather than dispensing lifecycle instructions, this endpoint will (after my branch https://github.com/GPII/universal/pull/549 is merged in) dispense a "preference-aligned pre-image of settings" or alternatively "settings in the ontology of preferences". I believe that this branch, aligned with Kasper's rollback work, dispenses "untransformed lifecycleInstructions", that is, payloads aligned with settingsHandler payloads but minus the application of the transformer - that is, within these payloads appear preferences in the flat common terms ontology. I hope that that is now perfectly clear :)